README.md

```diff
@@ -1 +1,21 @@
-# mesh-connect
\ No newline at end of file
+# Mesh Connect
+Mesh Connect is a project that aims to deliver a low cost communication infrastructure to areas that have no Internet connectivity at all. The project relies on low cost ESP32 devices with WiFi and LoRa radios. The role of WiFi connectivity is to allow any smartphone or PC to consume the Internet service provided by the ESP32 devices, which will form a mesh network through LoRa, being capable of deliver data over long distances with no need of a expensive infrastructure. Naturally, LoRa radios does not provide a large network throughput, but it is enought to allow basic texting applications to be used by the end users that live in areas with no Internet connectity at all.
+
+## Topology overview
+Each ESP32 that joins the Mesh Connect network topology may assume one of two roles:
```

Comment:
change "may" for "can" because we're talking about something that will definetely happen